deploy/kubernetes/base/kustomization.yaml

```diff
@@ -8,4 +8,4 @@ resources:
 images:
 - name: controller
   newName: ghcr.io/hewlettpackard/lustre-csi-driver
-  newTag: latest
+  newTag: 0.0.3
```

Comment:
Is there anyway with kustomize that we can set this value once somewhere?

Reply:
This is that "once somewhere" location for kustomize.

The other two spots are the baked output of "kustomize build <overlay>", which can then be used by "kubectl apply -f http:...".  We did these so that the user doesn't have to checkout this repo to install lustre-csi-driver.

I'd like to get rid of those baked outputs, and use "kubectl apply -k http:...", but first I need a solution to this "latest vs version" issue.